layouts/html.hbs

```diff
@@ -24,8 +24,12 @@
     <link rel="dns-prefetch" href="//dynl.mktgcdn.com">
     <link rel="dns-prefetch" href="//dynm.mktgcdn.com">
```

Comment:
do we know if these domains will be different @EmilyZhang777 ?

Reply:
I'm not sure. Based on [this](https://yext.slack.com/archives/CNVUY09HS/p1679068294340899) conversation, seems like eu is going to another bucket?